infrastructure/transform/feature-data-store.tf

```diff
@@ -60,21 +101,63 @@ resource "azurerm_mssql_database" "feature_database" {
   tags                 = var.tags
 }
 
-# AAD App + SPN for Databricks -> SQL Access.
+resource "null_resource" "create_sql_user" {
+  # make sure we have the identity created with aad perms, a client + secret to connect with, and an identity to set as dbo
+  depends_on = [
+    azuread_application_password.flowehr_sql_owner,
+    azuread_application.flowehr_databricks_sql,
+    azuread_app_role_assignment.sql_user_read_all,
+    azuread_app_role_assignment.sql_groupmember_read_all,
+    azuread_app_role_assignment.sql_application_read_all
+  ]
+
+  triggers = {
+    app_name      = azuread_application.flowehr_databricks_sql.display_name
+    database_name = azurerm_mssql_database.feature_database.name
+  }
+
+  # set the databricks 'user' app as dbo on the flowehr database
+  # load a csv file into a new SQL table
+  provisioner "local-exec" {
+    command = <<EOF
+      python ../../scripts/sql/create_sql_user.py $sql_server $database $app_id $app_secret $app_name
+      python ../../scripts/sql/load_csv_data.py $sql_server $database $app_id $app_secret "../../scripts/sql/nhsd-diabetes.csv" "diabetes"
+    EOF
+    environment = {
+      sql_server = azurerm_mssql_server.sql_server_features.fully_qualified_domain_name
+      database   = azurerm_mssql_database.feature_database.name
+      app_id     = azuread_application.flowehr_sql_owner.application_id
+      app_secret = azuread_application_password.flowehr_sql_owner.value
+      app_name   = local.databricks_app_name
```

Comment:
why both as env vars and cli args?
fwiw I'd vote env vars for consistency with the other python scripts

Reply:
yea ok, switched to env vars entirely, i guess you could still pass them in the cmd line if you want to call the script locally